docs/datatable-tutorial/simple_table.md

```diff
@@ -0,0 +1,104 @@
+```python exec
+import reflex as rx
+from pcweb.templates.docpage import docdemobox, doclink
+from pcweb.base_state import State
+from pcweb.pages.docs.datatable_tutorial.states import DataTableState, DataTableState2
+```
+
+
+
+# Data Table Tutorial
```

Comment:
If this isn't for the `rx.data_table` component, maybe the heading should be "Data Editor Tutorial"?

Reply:
yes this is something I discussed with nikhil, it is just that more commonly people refer to this component as a data table. Perhaps we could call it the data table (editable).